x/relationships/types/genesis.go

```diff
@@ -31,3 +39,14 @@ func ValidateGenesis(data *GenesisState) error {
 
 	return nil
 }
+
+// containDuplicates tells whether the given relationships slice contain duplicates of the provided relationship
+func containDuplicates(relationships []Relationship, relationship Relationship) bool {
+	var count = 0
+	for _, r := range relationships {
+		if r.Equal(relationship) {
+			count++
```

Comment:
I don't understand why you count the duplicates if you're not using that number later